src/framework/ui/topNavigation/topNavigation.component.tsx

```diff
@@ -135,12 +131,26 @@ export class TopNavigationComponent extends React.Component<TopNavigationProps>
 
   static styledComponentName: string = 'TopNavigation';
 
+  private getAlignmentDependentStyles = (): StyleType | null => {
+    const { alignment } = this.props;
+
+    if (alignment === 'start') {
```

Comment:
return by default instead of returning null

Reply:
Done.